src/lyapunovspectrum.jl

```diff
@@ -87,24 +87,25 @@ function modifiedGS!(A, Q, R, aⱼ, qᵢ, vⱼ)
 end
 
 """
-    liap_jetcoeffs!(eqsdiff!, t0, x, dx, xaux, δx, dδx, jac, vT)
+    lyap_jetcoeffs!(eqsdiff!, t, x, dx, xaux, δx, dδx, jac, _δv)
 
-Similar to [`jetcoeffs!`](@ref) for the calculation of the Liapunov
-spectrum. `jac` is the linearization of the equations of motion,
-and `xaux`, `δx` and `dδx` are auxiliary vectors.
+Similar to [`jetcoeffs!`](@ref) for the calculation of the Lyapunov spectrum.
+`jac` is the current value of the linearization of the equations of motion
+(i.e., the Jacobian), and `xaux`, `δx`, `dδx` and `_δv` are auxiliary vectors.
+The current value of the Jacobian is computed via automatic differentiation using
+`TaylorSeries.jl`.
 
 """
-function liap_jetcoeffs!(eqsdiff!, t::Taylor1{T}, x::Vector{Taylor1{U}},
+function lyap_jetcoeffs!(eqsdiff!, t::Taylor1{T}, x::Vector{Taylor1{U}},
```

Comment:
I think now `lyap_jetcoeffs!` is not used at all, so we should simply delete it. Note that some docstrings reference to this function, so they will have to be updated.